src/Screen/DetailsScreen/DetailsScreenView.tsx

```diff
@@ -1,30 +1,71 @@
-import React, { FC } from 'react';
-import { View, Text, FlatList } from 'react-native';
+import React, { FC, useState } from 'react';
+import { View, Text, FlatList, TouchableOpacity } from 'react-native';
 import { createStyles } from './style';
 import { DetailsScreenViewProps } from './types';
 import { useThemeAwareObject } from '@theme/ThemeAwareObject.hook';
 import { useTranslation } from 'react-i18next';
 import { faStar } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
 import { useTheme } from '@theme/Theme.context';
+import { useAppDispatch, useAppSelector } from '@redux/hooks';
+import { addFavoriteAirport, deleteFavoriteAirport } from '@redux/action/favoriteAirpots';
+import { AirportsListTypes } from '@redux/api/type';
+import { getFavoriteAirport } from '@redux/selectors';
 
 export const DetailsScreenView: FC<DetailsScreenViewProps> = ({
   airportData,
   linksAboutAirport,
   renderItem,
 }) => {
+  const isActive = () =>
+    favoriteAirports.find((item) => {
```

Comment:
please use return instead of => (). 

() => {
return `something`
}

Reply:
and let's move that function a bit below, cause more clear and readable write useStates and other hooks first in the components.